pkg/iac-providers/cft/v1/load-file.go

```diff
@@ -0,0 +1,94 @@
+/*
+    Copyright (C) 2020 Accurics, Inc.
+
+	Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+		http://www.apache.org/licenses/LICENSE-2.0
+
+	Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+
+package cftv1
+
+import (
+	"fmt"
+	"io/ioutil"
+	"strings"
+
+	"github.com/accurics/terrascan/pkg/iac-providers/output"
+	"github.com/accurics/terrascan/pkg/mapper"
+	"github.com/accurics/terrascan/pkg/utils"
+	"go.uber.org/zap"
+)
+
+// LoadIacFile loads the specified CFT template file.
+// Note that a single CFT template json file may contain multiple resource definitions.
+func (a *CFTV1) LoadIacFile(absFilePath string) (allResourcesConfig output.AllResourceConfigs, err error) {
+	var iacDocuments []*utils.IacDocument
+	fileExt := a.getFileType(absFilePath)
+	switch fileExt {
+	case YAMLExtension:
+		fallthrough
+	case YAMLExtension2:
+		iacDocuments, err = utils.LoadYAML(absFilePath)
```

Comment:
`utils.LoadYAML` is not very useful here, because the number of iac documents is always going to be 1.
Also, using this function is adding extra code as it removes the intrinsic tags.

Reply:
I agree with @patilpankaj212. It is not necessary to use `utils.LoadYAML()`. In this particular case. The file data is being parsed multiple number of times which redundant and unnecessary. 

We can implement our own loader and optimize.